doc/api/tls.md

```diff
@@ -1260,6 +1260,7 @@ changes:
     provided.
     For PEM encoded certificates, supported types are "TRUSTED CERTIFICATE",
     "X509 CERTIFICATE", and "CERTIFICATE".
+    See also [`tls.roots`].
```

Comment:
Missing bottom reference?

Reply:
Oops, that got lost somehow, somewhere. Fixed now.